sites/partners/src/components/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -144,6 +152,20 @@ const BuildingDetails = ({
       : defaultMessage
   }
 
+  const enableRegions = doJurisdictionsHaveFeatureFlagOn(
+    FeatureFlagEnum.enableRegions,
+    listing?.jurisdictions?.id
+  )
```

Comment:
Issue: This isn't working as expected. I have the "Bloomington" jurisdiction set to true and another jurisdiction where it is not turned on. No matter what jurisdiction you have selected on a new listing the enableRegions is showing. The only way I can get it to disappear is to save it with the other jurisdiction and then open the listing to edit it.

Potential solution would be to switch out `listing?.jurisdictions?.id` with the jurisdiction id from the form. This can be grabbed with `const jurisdiction = watch("jurisdictions.id")`. `AdditionalFees.tsx` is a good example of it